packages/core/src/physics/shape/ColliderShape.ts

```diff
@@ -66,13 +66,16 @@ export abstract class ColliderShape implements ICustomClone {
   }
 
   /**
-   * Physical material.
+   * Physical material, material can't be null.
    */
   get material(): PhysicsMaterial {
     return this._material;
   }
 
   set material(value: PhysicsMaterial) {
+    if (!value) {
+      throw new Error("The physics material of the shape can't be null.");
+    }
```

Comment:
`This._material` is null when initialized. Do you mean that the initialized value is illegal?
Why not 
```
set material(value: PhysicsMaterial) {
    if (this._material !== value) {
      this._material = value;
      value && this._nativeShape.setMaterial(value._nativeMaterial);
    }
  }
```

Reply:
The shape is created with a default material